### PR TITLE
DBZ-4471 Introducing Debezium Server Example with MySQL and GCP PubSub

### PR DESCRIPTION
**Pull request by ilyasahsan123**
https://issues.redhat.com/browse/DBZ-4471

### CODE REVIEW

File: debezium-server/debezium-server-sink-pubsub/config-mysql/application.properties [hunk 1]
```diff
@@ -0,0 +1,12 @@
+debezium.sink.type=pubsub
```
Comment (jpechane) [inline debezium-server/debezium-server-sink-pubsub/config-mysql/application.properties:1]: Did you try to run it? Does it really work without internal schema history configured?

**Comment by jpechane** [comment]
@ilyasahsan123 Thanks for the PR, could you please remove the brackets around `DBZ-4471` in commit messages?

File: debezium-server/debezium-server-sink-pubsub/config-mysql/application.properties (hunk 1 above)
Comment (ilyasahsan123) [reply debezium-server/debezium-server-sink-pubsub/config-mysql/application.properties:1]: Edited. I also tested the connector with the result event was successfully sent to Pubsub.

**Comment by ilyasahsan123** [comment]
> @ilyasahsan123 Thanks for the PR, could you please remove the brackets around `DBZ-4471` in commit messages?

Sure. edited.

File: debezium-server/debezium-server-sink-pubsub/README.md [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+```
+
+Test the setup by making changes to the customers table. The logs will appear in Google Cloud Pub/Sub shortly. You can access Postgres with this command:
+
+```shell
+docker compose exec postgres env PGOPTIONS="--search_path=inventory" bash -c 'psql -U $POSTGRES_USER postgres'
+```
+
+
+### MongoDB Debezium Connector
```
Comment (jpechane) [inline debezium-server/debezium-server-sink-pubsub/README.md:85]: Please take a look at original MongoDB documentation. The setup sequence is different from MySQL or Postgres. Also this text speaks about database and table which is not correct in context of MongoDB.

**Review by jpechane — CHANGES_REQUESTED** [review]
@ilyasahsan123 Thanks a lot. IMHO we are almost done. There is just one remaining issue in MongoDB documentation.

File: debezium-server/debezium-server-sink-pubsub/README.md (hunk 2 above)
Comment (ilyasahsan123) [reply debezium-server/debezium-server-sink-pubsub/README.md:85]: Thanks for your feedback,@jpechane. Could you please re-review this?

**Comment by jpechane** [comment]
@ilyasahsan123 Applied, thanks a lot

**Pull request merged**